### PR TITLE
automl: move video classification samples out of branch

### PR DESCRIPTION
**Pull request by nnegrey**

### CODE REVIEW

**Comment by nnegrey** [comment]
@dmahugh @sirtorry friendly WFH ping (no rush)

File: automl/beta/video_classification_create_model.py [hunk 1]
```diff
@@ -12,21 +12,21 @@
...
+def create_model(
+    project_id="YOUR_PROJECT_ID",
+    dataset_id="YOUR_DATASET_ID",
+    display_name="your_models_display_name",
+):
+    """Create a automl video classification model."""
     client = automl.AutoMlClient()
 
-    # A resource that represents Google Cloud Platform location.
+    # A resource that represents Google Cloud Platform loacation.
```
Comment (telpirion) [inline automl/beta/video_classification_create_model.py:27]: Typo?

Comment (leahecole) [reply automl/beta/video_classification_create_model.py:27]: GOOD EYES

Comment (nnegrey) [reply automl/beta/video_classification_create_model.py:27]: lol

**Pull request merged**